mon-pix/app/services/url.js

```diff
@@ -1,12 +1,19 @@
 import Service  from '@ember/service';
 import { inject as service } from '@ember/service';
+import ENV from '../config/environment';
 
 const FRENCH_DOMAIN_EXTENSION = 'fr';
 
 export default class Url extends Service {
   @service currentDomain;
+  definedHomeHost = ENV.APP.HOME_HOST;
 
   get isFrenchDomainExtension() {
     return this.currentDomain.getExtension() === FRENCH_DOMAIN_EXTENSION;
   }
+
+  get homeUrl() {
+    const homeUrl = `https://pix.${this.currentDomain.getExtension()}`;
+    return this.definedHomeHost || homeUrl;
+  }
```

Comment:
Pourquoi on distingue homeUrl et homeHost ? Ça me perturbe pas mal de retourner un host quand on get un url

```suggestion
  get homeUrl() {
    const homeUrl = `https://pix.${this.currentDomain.getExtension()}`;
    return this.definedHomeUrl || homeUrl;
  }
```

ou full-host ou même carrément 🤷
```suggestion
  get home() {
    const homeUrl = `https://pix.${this.currentDomain.getExtension()}`;
    return this.definedHomeHost || homeUrl;
  }
```

Reply:
L'intention est bien de retourner sur l'url de la home.
Il se trouve que si on définit un `HOME_HOST` alors on l'url de la home est `HOME_HOST`.

Reply:
`get homeUrl()`: ça reste clair que l'on retourne une url.
La première solution proposée par Stéphane me semble pas mal, mais dans ce cas faudrait-il aussi renommer la variable d'environnement `HOME_URL` plutôt que `HOME_HOST` ?

Reply:
le homeUrl me parait important de le laisser ainsi.
pour le HOME_HOST et definedHomeHost je comprend l'incompréhension possible, si on change pour definedHomeUrl je changerais aussi le HOME_HOST en HOME_URL

Reply:
ok je change.

Reply:
Je suis ok avec `homeUrl` mais la nature de `HOME_HOST` est bien un host et non une url

Reply:
il s'agit bien d'une URL : la variable vaut `/` sur les review apps